react/src/pages/participants/ParticipantTable.tsx

```diff
@@ -69,15 +71,18 @@ export default function ParticipantTable() {
                 );
             }
         });
-    }, []);
+    }, [num]);
 
     return (
         <div>
             {activeRow && (
                 <ParticipantInfoDialog
                     open={detail}
                     participant={activeRow}
-                    onClose={() => setDetail(false)}
+                    onClose={() => {
+                        setDetail(false);
+                        reRender(n => n + 1);
```

Comment:
Same as the dataset table, a re-render should only occur if the user changed something.